api/v1alpha1/nonadminbackup_types.go

```diff
@@ -51,9 +51,9 @@ type VeleroBackup struct {
 	// +optional
 	Status *velerov1.BackupStatus `json:"status,omitempty"`
 
-	// name references the Velero backup by it's name.
+	// nabnacuuid references the Non Admin Backup object by it's nacUUID.
 	// +optional
-	Name string `json:"name,omitempty"`
+	NabNacUUID string `json:"nabnacuuid,omitempty"`
```

Comment:
I think we should keep the VB Name as well as the UUID.

Reply:
In our internal implementation the VB Name is same as UUID, so I am unsure if we want to duplicate that at this point of time? I am happy to add it if you think so:

https://github.com/migtools/oadp-non-admin/blob/97dacc63f1f009c9b6422b02165438614d482a29/internal/controller/nonadminbackup_controller.go#L251

Also please not my comment on:
https://github.com/migtools/oadp-non-admin/issues/90#issuecomment-2412374910

I think we could close 90 as well with this if we agree that retry is not required anymore (almost impossible collision in our case).

Reply:
Hmm I was thinking this field should reflect both the Name and UUID. Lets rename it to something like: `identifier` , `nameUUID` or `idName` or something.

Reply:
ok, works for me.